src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,11 +1,83 @@
 package core.basesyntax;
 
 /**
- * <p>Дано класс Plane. Написать класс PlaneBuilder, который реализует паттерн Builder.
+ * <p>Дано класс Plane. Написать класс Builder, который реализует паттерн Builder.
  * Добавьте не менее пяти полей в класс Plain. Внести все необходимые изминения в класс Plane
  * которые могут потребоваться для реализации паттерна Builder.</p>
  */
 
 public class Plane {
+    private String planeType;
+    private int numberOfPassengers;
+    private String yearOfProduction;
+    private String whichAirlines;
+    private int code;
 
+    private Plane(Builder builder) {
```

Comment:
@RomanHreits please add the Main class with an example of how to use your builder

Reply:
done